firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionEvents.kt

```diff
@@ -80,4 +86,13 @@ internal object SessionEvents {
         )
     )
   }
+
+  private fun toDataCollectionState(subscriber: SessionSubscriber?): DataCollectionState =
```

Comment:
Is the subscriber optional?

Reply:
Yeah. If it's not present, then it means that sdk is not installed.